packages/react-dev-utils/eslintFormatter.js

```diff
@@ -18,59 +18,59 @@ function isError(message) {
 }
 
 function formatter(results) {
-  let output = '\n';
   let hasErrors = false;
   let reportContainsErrorRuleIDs = false;
 
-  results.forEach(result => {
-    let messages = result.messages;
+  let output = results.reduce((finalString, result) => {
+    let { messages } = result;
```

Comment:
This won't work in Node@4, but it looks like CRA has dropped support for that.

Reply:
should I stop using destructuring then?

Reply:
Other parts of the codebase already use features that aren't supported in Node 4 and tests don't even run for that version, I guess its safe to keep using it. :)